indico/modules/categories/controllers/base.py

```diff
@@ -57,3 +57,18 @@ class RHManageCategoryBase(RHCategoryBase):
     def _checkProtection(self):
         if not self.category.can_manage(session.user):
             raise Forbidden
+
+
+class RHMoveCategoryBase(RHManageCategoryBase):
+    def _checkParams(self):
+        RHManageCategoryBase._checkParams(self)
+        target_category_id = request.form.get('target_category_id')
+        if target_category_id is None:
+            self.target_category = None
+        else:
+            self.target_category = Category.get(int(target_category_id), is_deleted=False)
```

Comment:
do you ever expect an invalid ID here? if not use `get_one` instead; that will fail nicely if there was no match